outputs.tf

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-output backend_services {
+output "backend_services" {
   description = "The backend service resources."
-  value       = "${google_compute_backend_service.default.*.self_link}"
+  value       = [for i in range(length(google_compute_backend_service.default)) : google_compute_backend_service.default[i].self_link]
```

Comment:
This still needs to be resolved.

Reply:
@aaron-lane, thank you for review! [Fixed](https://github.com/terraform-google-modules/terraform-google-lb-http/compare/6413deded338c73c79126d468a8a869bb9257eb5..01cd4b73e3776eae62046bd310440cba4fb3eeef)

Originally I tried that way but it didn't work, probably because of lack of some other optimization or left-overs from for_each migration. 
Retested all the examples(except for shared_vpc) - no issues, test is passing.